akka-stream/src/main/scala/akka/stream/impl/streamref/StreamRefsMaster.scala

```diff
@@ -23,9 +23,7 @@ private[stream] object StreamRefsMaster extends ExtensionId[StreamRefsMaster] wi
 
 /** INTERNAL API */
 @InternalApi
-private[stream] final class StreamRefsMaster(system: ExtendedActorSystem) extends Extension {
-
-  private val log = Logging(system, getClass)
+private[stream] final class StreamRefsMaster(@unused system: ExtendedActorSystem) extends Extension {
```

Comment:
Since it's private, can we remove the `system` parameter?

Reply:
I had some bad luck when I was thinking about how extensions are instantiatied, yes, we should be able to remove that.